fmriprep/viz/report.tpl

```diff
@@ -82,10 +83,43 @@ body {
             {% endfor %}
             {% endif %}
         {% endfor %}
-
     {% endif %}
     </div>
 {% endfor %}
 
+<div id="errors">
+    <h1 class="sub-report-title">Errors</h1>
+    <ul>
+    {% for error in errors %}
+        <li>
+        <div class="nipype_error">
+            Node Name: <a href="#" onclick="toggle('{{error.file|replace('.', '')}}_details_id');">{{ error.node }}</a><br>
+            <div id="{{error.file|replace('.', '')}}_details_id" style="display:none">
+            File: {{ error.file }}<br>
+            Working Directory: {{ error.node_dir }}<br>
+            Inputs: {{ error.inputs }}<br>
```

Comment:
Isn't `inputs` a Bunch object? Or a traited class? If so you should be able to iterate over it fields and format inputs one by one. This might be helpful to iterate over traited spec: https://github.com/nipy/nipype/blob/bbc93ae5f4efee89a2d61541fbce6d8d7ab779a5/nipype/interfaces/base.py#L874